tests/Feature/FlightControllerTest.php

```diff
@@ -2,547 +2,570 @@
 
 namespace Tests\Feature;
 
-use App\Http\Controllers\FlightController;
+use App\Actions\Flight\UpdateFlightAction;
+use App\Http\Controllers\Flight\DeleteFlightController;
+use App\Http\Controllers\Flight\GetFlightController;
+use App\Http\Controllers\Flight\StoreFlightController;
+use App\Http\Controllers\Flight\UpdateFlightController;
 use App\Models\Airline;
 use App\Models\City;
 use App\Models\Flight;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Tests\TestCase;
 
-class FlightControllerTest extends TestCase
-{
-    use RefreshDatabase;
-
-    /** @test */
-    public function testGetFlightsForAnAirline()
-    {
-        City::factory(2)->create();
-
-        Airline::factory()->create();
-
-        Flight::create([
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
+it('gets flights for an airline ', function () {
+    $origin = City::factory()->create();
+    $destination = City::factory()->create();
+
+    $airline = Airline::factory()->create();
+
+    Flight::create([
```

Comment:
we should use factories for flights as well. The idea is we only pass the pertinent data we actually care about in the test, and let the factory invent the rest. In this case, if a new random field appears on flight that we do not care about in here, the test might fail becuase we are not passing it to the create, while using the Factory would have prevented the issue